docs/extend/modular-resources/key-concepts.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Key Concepts of Modular Resource APIs"
+linkTitle: "Concepts"
+weight: 10
+type: "docs"
+tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
+description: "The key concepts behind how Viam's resource APIs and models are uniquely namespaced and modular resource management with the RDK."
+no_list: true
+---
+
+`viam-server` [manages](#management) modular {{< glossary_tooltip term_id="resource" text="resources" >}} configured on your robot like resources that are already built into the Robot Development Kit [(RDK)](/internals/rdk/).
```

Comment:
```suggestion
`viam-server` [manages](#management) modular {{< glossary_tooltip term_id="resource" text="resources" >}} configured on your robot the same way it manages resources that are already built into the Robot Development Kit [(RDK)](/internals/rdk/).
```
To make clear that "like" doesn't mean "such as"

Reply:
Mostly just as the sentence has been up for a while and isn't new